lazy_tensor_core/lazy_tensor_core/csrc/debug_util.cpp

```diff
@@ -102,7 +102,7 @@ std::string DebugUtil::GetTensorsGraphInfo(c10::ArrayRef<LazyTensor> tensors,
     graph_str = ir::DumpUtil::ToDot(root_nodes);
   } else if (format == GraphFormat::kBackend) {
     graph_str = ir::DumpUtil::ToBackend(
-        root_values, unique_device ? *unique_device : GetCurrentDevice());
+        root_values, unique_device ? *unique_device : Device());
```

Comment:
I think it's a little messy to replace a 'current device' function with calling the default device ctor.  Is your plan that later on, the Device() ctor will actually query the backend for an appropriate default?  or does Device() leave type_ uninitialized and the surrounding code is supposed to understand that to mean no setting has been set yet?

Reply:
Yea, I added a TODO for the default ctor of the device to query the default device type or default device in the future. Let me add a comment in the default ctor to make it more clear.